src/sentry/api/endpoints/chunk_upload.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+# from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry.models import FileBlob
+from sentry.models.file import DEFAULT_BLOB_SIZE
+from sentry.api.base import Endpoint
+from sentry.api.bases.project import ProjectReleasePermission
+
+
+UPLOAD_ENDPOINT = 'https://sentry.io'
```

Comment:
Why is this a hardcoded value? This isn't going to work for on-premise.

Reply:
Right, is there a way to get the current URL or something?
We were thinking about not serving the URL of the installation instead serve `upload.sentry.io` which handles stuff a bit differently.

Reply:
Yeah, the `system.url-prefix` `option`. I think we should introduce a new option here so it can be overridden in config. But default back to `system.url-prefix`.